client/src/Movies/MovieList.js

```diff
@@ -34,21 +39,8 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
```

Comment:
After your refactor, this line is no longer necessary because you are bringing in { movie } up top and simply passing that entire movie object to the MovieCard.. it doesn't break your code by any means though!